bliss/conf/base_config.yaml

```diff
@@ -161,16 +161,18 @@ surveys:
             mean_sources: 0.2
             min_sources: 0
             prob_galaxy: 0.72
-            star_flux_min: 622
-            star_flux_max: 1e6
+            star_flux_min: 0.63094948
+            star_flux_max: 1014
```

Comment:
Instead of making these values based on the conversion from 94/1/12, I think it would be better to just use some general values like 0.6 - 1500 or something

Reply:
These are actually from 94, 125, 752 and 3900. I used the same values that we had been using (in units of electron counts) and just divided by the average r_band ratio across all of the above runs.

Reply:
Either way seems OK to me. Maybe better to keep them exact for now so that if our results change following the PR, we'll know it wasn't because of rounding these values.